test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -360,6 +360,19 @@ TEST_VM(os, jio_snprintf) {
 //  (from multiple calls to os::reserve_memory)
 static address reserve_multiple(int num_stripes, size_t stripe_len) {
   assert(is_aligned(stripe_len, os::vm_allocation_granularity()), "Sanity");
+
+#ifdef __APPLE__
+  // Workaround: try reserving memory with executable=true
+  // to figure out if that's supported on this macOS version
+  const bool executable = true;
+  size_t test_len = 128;
```

Comment:
Just curious: since you made local var `executable` `const`, I have to wonder
why you didn't make local var `test_len` `const`?

Reply:
copy/paste consequence, did not meant to make it `const`, but copied the line from the existing code that used it. I will remove it.